source/dub/internal/colorize/cwrite.d

```diff
@@ -0,0 +1,74 @@
+/**
+ * Authors: ponce
+ * Date: July 28, 2014
+ * License: Licensed under the MIT license. See LICENSE for more information
+ * Version: 1.0.2
+ */
+module dub.internal.colorize.cwrite;
```

Comment:
This entire module is only used once by `winterm`.
I think it would be best to move only the relevant code to `winterm` and delete the rest.

Reply:
I didn't get this one, do you mean that this module uses `winterm` only once? (i.e. not the other way around)? How would you propose to adjust the dub.internal.colorize subpackage? At this point it wouldn't even make a lot of sense to keep it under there, since some stuff will be removed (some sub modules too, right?)

Should I simply make the changes you proposed and leave everything there (but then it would seem like I "imported" d-colorize but changed the library from top to bottom, which would be a bit strange) or make a new module somewhere else? And how should that be called/where should it be?

Reply:
Oh I meant that winterm only calls one method from this module, so there's a lot of dead code that could simply be removed?

> Should I simply make the changes you proposed and leave everything there (but then it would seem like I "imported" d-colorize but changed the library from top to bottom, which would be a bit strange) 

Ah I didn't know that you simply copied the library in. I don't think that modifying this code is a problem, especially because I doubt that we will ever need to merge new features from this library. So I don't see a problem with removing dead code and making it more light-weight if you are interested in going in this direction.